### PR TITLE
Other API's sample code

### PR DESCRIPTION
**Pull request by dabsun01**

### CODE REVIEW

File: code_samples/CSharp_samples/SampleClientCode/FieldsClient.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+
+        public async Task<string> SendField()
+        {
+            var fieldApiUrl = _resourcesApiUrl + "/" + _orgId +  "/fields";
+            using var httpClient = new HttpClient();
+
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _accessToken);
+
+            //change the field name every time you make a request as the filed api does not allow duplicate names
```
Comment (haydencbarnes) [inline code_samples/CSharp_samples/SampleClientCode/FieldsClient.cs:28]:
```suggestion
            //change the field name every time you make a request as the field api does not allow duplicate names
```

File: code_samples/CSharp_samples/SampleClientCode/BoundariesClient.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        {
+            var boundariesApiUrl = _resourcesApiUrl + "/" + _orgId + "/boundaries";
+
+            using var httpClient = new HttpClient();
+
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _accessToken);
+
+            var boundaryToCreate = new Boundary()
+            {
+                Name = "Some Test Boundary15",
```
Comment (haydencbarnes) [inline code_samples/CSharp_samples/SampleClientCode/BoundariesClient.cs:35]: Add: // Add your boundary info here

File: code_samples/CSharp_samples/SampleClientCode/GuidanceLinesClient.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        public async Task<string> SendGuidanceLines()
+        {
+            var guidanceLinesApiUrl = _resourcesApiUrl + "/" + _orgId + "/guidancelines";
+            using var httpClient = new HttpClient();
+
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _accessToken);
+            
+            var guidanceLineToCreate = new GuidanceLine()
+            {
+                Name = "Some Guidance Line15",
```
Comment (haydencbarnes) [inline code_samples/CSharp_samples/SampleClientCode/GuidanceLinesClient.cs:35]: Add: // Add your guidance line info here

**Review by haydencbarnes — CHANGES_REQUESTED** [review]

File: code_samples/CSharp_samples/SampleClientCode/FieldsClient.cs (hunk 1 above)
Comment (dabsun01) [reply code_samples/CSharp_samples/SampleClientCode/FieldsClient.cs]: fixed

File: code_samples/CSharp_samples/SampleClientCode/BoundariesClient.cs (hunk 2 above)
Comment (dabsun01) [reply code_samples/CSharp_samples/SampleClientCode/BoundariesClient.cs:36]: fixed

File: code_samples/CSharp_samples/SampleClientCode/GuidanceLinesClient.cs (hunk 3 above)
Comment (dabsun01) [reply code_samples/CSharp_samples/SampleClientCode/GuidanceLinesClient.cs:36]: fixed

**Review by haydencbarnes — APPROVED** [review]

**Pull request merged**